.github/workflows/makeRelease.yml

```diff
@@ -32,7 +37,41 @@ jobs:
           asset_path: /opt/code/clib4-${{ github.event.release.tag_name }}.lha
           asset_name: clib4-${{ github.event.release.tag_name }}.lha
           asset_content_type: application/x-lzh-compressed
-      # - name: Release on OS4Depot
-      # - name: Release on Aminet
-      # - name: Prepare Ubuntu apt packages
-      # - name: Release on Ubuntu apt server
\ No newline at end of file
+      - name: Upload DEB release file in GitHub
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: /opt/code/clib4-${{ github.event.release.tag_name }}_amd64.deb
+          asset_name: clib4-${{ github.event.release.tag_name }}_amd64.deb
+          asset_content_type: application/vnd.debian.binary-pac
+      - name: Upload DEB release file to the server
+          uses: kostya-ten/ssh-server-deploy@v4
+          with:
+            host: ${{ secrets.DEBSERVER_HOST }}
+            port: ${{ secrets.DEBSERVER_PORT }}
+            username: ${{ secrets.DEBSERVER_USERNAME }}
+            private_key: ${{ secrets.DEBSERVER_PRIVATE_KEY }}
+            scp_source: clib4-${{ github.event.release.tag_name }}_amd64.deb
+            scp_target: /opt/amigarepo/ubuntu/pool/main
+            before_script: |
+              rm /opt/amigarepo/ubuntu/pool/main/clib4*.deb
+            after_script: |
+              /root/regenerate-packages.sh
```

Comment:
This script generates the packages files that are needed for the apt to access them. This is needed since we upgraded the clib4 file